knowledge-base/reportviewer-styles-and-images-messed-up.md

```diff
@@ -0,0 +1,60 @@
+---
+title: ReportViewer Styles and Images Are All Messed Up
+page_title: The ReportViewer Styles and Images Are All Messed Up
+description: "Learn how to fix the rendering of the styles and images in the ReportViewer when working with Telerik Reporting."
+slug: reportviewer-styles-images-messedup
+tags: telerik, reporting, run, time, reportviewer, styles, and, images, messed, up, rendering
+previous_url:
+published: True
+component:
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
```

Comment:
Related only to the legacy AAP.NET WebForms Viewer

Include a note about this so that it is clear to the readers. Most of them don't use this viewer anymore and we recommend them to migrate to the Html5 Viewer